src/pudl/output/pudltabl.py

```diff
@@ -127,9 +126,6 @@ def __init__(
         self.fill_tech_desc = fill_tech_desc  # only for eia860 table.
         self.unit_ids = unit_ids
 
-        # Used to persist the output tables. Returns None if they don't exist.
-        self._dfs = defaultdict(lambda: None)
-
```

Comment:
We don't actually cache dataframes in PudlTabl any more. Not sure why this was sticking around.